docusaurus/docs/iOS/components/message.md

```diff
@@ -1,3 +1,199 @@
 ---
 title: Message
----
\ No newline at end of file
+---
+
+The `ChatMessageListVC` component renders the list of messages using UIKit TableView. Messages rendering is delegated to different classes that can be customized or swapped with a custom class.
```

Comment:
I don't think it is worth it to mention that we use UITableView for this. It is an implementation detail, and it will most likely change in the future. (At some point we want to go back to CollectionView since it provides more flexibility)

Reply:
I think its not completely useless to know, easy to change it as well in the future :)